### PR TITLE
Create core-html-elements-discussion-insight.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
footnotes might need updating

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| core-html-elements-discussion-insight | No Enki username has been found for this author. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/core-html-elements/core-html-elements-discussion-insight.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/core-html-elements/core-html-elements-discussion-insight.md?ref=9ea4dc12fee2fc9497158c15d2f85b8fefdeab5a)

File: web/html/core-html-elements/core-html-elements-discussion-insight.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Can you write HTML code for a webpage that would display:
+- The largest heading containing the title of the page
+- An image of any size. To be nice to your viewers add an alternative text message in case the image cannot load.
+- One paragraph with information about the page.
+- A link to another website
+
+It's up to you to choose the title, the image, what the contents of those paragraphs will be, as well as their color.
+
+Here's one example of how the page can look:
```
Comment (nem035) [inline web/html/core-html-elements/core-html-elements-discussion-insight.md:29]: I think "like" is missing here?

```suggestion
Here's one example of how the page can look like:
```

**Review by nem035 — DISMISSED** [review]
:+1:

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
@Stefan-Stojanovic you forgot to add the insight to the workout's README

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
@Stefan-Stojanovic please delete `parent: meta-elements` on line 5 in the workout's README ([here](https://github.com/enkidevs/curriculum/pull/2471/files#diff-a2a76c783df3fa94ac8c1827af8003ab3db9d4d12df1f8233100ff2ce358de2cR5))

**Review by nem035 — DISMISSED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**